dvc/utils/__init__.py

```diff
@@ -270,6 +270,17 @@ def colorize(message, color=None):
     )
 
 
+def styled(message, style=None):
```

Comment:
@skshetry this awesome and long waited from my end ... the only question - how well it supported by older/SSH terminals, Windows, etc - we should do some fallback to regular colors? or will this logic handle this automatically?

(also, really wanted to improve the way we print the box around telemetry, updater messages - there was some concern with older terminals as well that prevented us from using nice symbols - may be we can detect it also)

Reply:
(I see that we only change brightness here, also as a good next step we can start using normal modern colors instead of RED, BLUE - etc that are even hard to read sometimes - again the biggest question is compatibility and fallbacks)

Reply:
It should work on Windows as it uses colorama, but not sure of others.

Reply:
I was using colors before, like I showed yesterday. But, I moved away from it, because if other log appears in between, mix of colored and non-colored output looks weird. Plus, the colors that work on both dark/light terminals need to be choosen (most probably, red/green/yellow).

Reply:
> But, I moved away from it, because if other log appears in between, mix of colored and non-colored output looks weird.

do you mean when we use `-v`?

> I was using colors before, like I showed yesterday. 

Sorry, I was confusing. Your are right about using brightness for these logs. Everything else I mentioned is not directly related to this PR. So, please feel free to disregard :)

I was saying about using "pastel" terminal colors instead of regular RED, BLUE - e.g. https://github.com/tartley/colorama/blob/master/colorama/ansi.py#L60 - for all other logs. Same as `nmp` or `heroku cli` do - I think they can detect if terminal supports modern colors or not.

Also using some nice looking symbols for the box instead of a regular dash.

Reply:
> do you mean when we use -v?

No. I mean, as we use INFO for ui, we can either support only one color (colored or not) or manually change all to whatever color we like. But, any `logger.info` call will not be colored, so we might end up with mixed output. To have a colored output, we'd constantly fighting with it. 

> Same as nmp or heroku cli do

I did gave a quick look to https://github.com/chalk/supports-color, looks like it's possible. But, do we really need True colors (i.e. 256 colors are enough for us? At least for a start).

Reply:
256 is totally enough. (again, not really related to this PR, I would suggest a ticket for this)

> No. I mean, as we use INFO for ui, we can either support only one color (colored or not) or manually change all to whatever color we like. But, any logger.info call will not be colored, so we might end up with mixed output. To have a colored output, we'd constantly fighting with it.

not sure I completely follow ... but looks like another reason to not use logger for UI.